test/e2e/e2e_test.go

```diff
@@ -767,6 +857,39 @@ func TestE2E(t *testing.T) {
 		}
 	})
 
+	// make sure that deleting service-cert-secret regenerates a working secret again
+	t.Run("serving-cert-secret-delete-data", func(t *testing.T) {
+		serviceName := "metrics"
+		namespace := "openshift-service-ca-operator"
```

Comment:
Given that there are 2 namespace vars in this closure, please differentiate them (e.g. `operatorNamespace`, `testNamespace`).

Reply:
Done